win32/tests/threadpool_win32_int/threadpool_win32_int.c

```diff
@@ -67,7 +69,7 @@ typedef struct OPEN_WORK_CONTEXT_TAG
 static void open_work_function(void* context)
 {
     OPEN_WORK_CONTEXT* open_work_context = (OPEN_WORK_CONTEXT*)context;
-    ASSERT_ARE_NOT_EQUAL(int, 0, threadpool_open(open_work_context->threadpool));
+    ASSERT_ARE_EQUAL(int, 0, threadpool_open(open_work_context->threadpool));
     (void)InterlockedIncrement(&open_work_context->call_count);
```

Comment:
What are we testing here with threadpool_open?
This should be completely removed from the threadpool tests with this PR.
There is no value in having threadpool_open and threadpool_close tested in threadpool_ut and threadpool_int.

Removing from all the other users will come with other PR(s), but after this PR they should not be tested anymore (as they do nothing).

Reply:
In UT, only the open and close function that remain is tested to satisfy Traceability tool.